lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -360,6 +361,42 @@ static LogicalResult verifyFModuleOp(FModuleOp module) {
   return success();
 }
 
+// TODO: This is a clone of mlir::FuncOp::eraseArguments, which could be moved
```

Comment:
I think we can move this helper from FuncOp to FunctionLike and then FModuleOp would have access to it. If that makes sense I can file an issue with MLIR and work on that change upstream. It's a pretty finicky function to get right, so I ended up just copying it for now. Is there a better way to get access to this function without modifying upstream MLIR? I thought about having FModuleOp be a subclass of FuncOp, but I don't think we want that.